### PR TITLE
Bugfix/ctv 1198 android tv fixes

### PR DESCRIPTION
**Pull request by rblaa**
as discussed, android tv fixes needed to get running in the emulator

### CODE REVIEW

File: src/focus_manager/txm_platform.js [hunk 1]
```diff
@@ -504,11 +510,16 @@ export class TXMPlatform {
                 self.model = details[1];
                 self.version = details[2];
             } else {
-                var detailSubstring = userAgent.substring(userAgent.indexOf("(") + 1, userAgent.indexOf(")"));
-                var detailSplit = detailSubstring.split(";");
-                self.model = detailSplit[2].split(" ").find(part => {return part});
-                const versionParts = detailSplit[1].split(" ");
-                self.version = versionParts[versionParts.length - 1];
+                var groupStart = userAgent.indexOf("(");
+                if (groupStart >= 0) {
```
Comment (colejona) [inline src/focus_manager/txm_platform.js:514]: What if this condition is false? is `self.model` and `self.version` not being defined a problem? Sorry -- can't dive too deep right now

Comment (rblaa) [reply src/focus_manager/txm_platform.js:514]: there is a default value of "Unknown" for them

Comment (rblaa) [reply src/focus_manager/txm_platform.js:514]: the .model assignment part is technically a bug, but is overridden by the callers to do something better for the actual platform

Comment (rblaa) [reply src/focus_manager/txm_platform.js:514]: so, let me know if you want a fix and a version bump, i.e. only the version should be set in this part

Comment (colejona) [reply src/focus_manager/txm_platform.js:514]: nah, just wanted to check if something was forgotten

**Review by colejona — APPROVED** [review]

**Pull request merged**